lib/gamnit/gamnit.nit

```diff
@@ -65,3 +67,6 @@ redef class App
 	# The implementation varies per platform.
 	private fun feed_events do end
 end
+
+if "NIT_TESTING".environ == "true" then exit 0
+super
```

Comment:
should be elsewere because the main program might not call `super` in its main.
Maybe you can redifine `Sys.run` instead